xikolo-ios/ViewControllers/Main/LoginViewController.swift

```diff
@@ -43,21 +43,23 @@ class LoginViewController : AbstractLoginViewController, WKUIDelegate {
     }
 
     @IBAction func singleSignIn(_ sender: UIButton) {
-        performSegue(withIdentifier: "ShowSSOWebView", sender: self)
-//        let url = URL(string: Routes.SSO_URL)!
-//        let request = URLRequest(url: url)
-//        let wkViewController = CustomHeaderWebView(frame: .zero)
-//        wkViewController.header = ["X-User-Platform" : "iOS"]
-//        wkViewController.uiDelegate = self
-//       
-//        view = wkViewController
-//        wkViewController.load(request)
+        self.performSegue(withIdentifier: "ShowSSOWebView", sender: self)
     }
 
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
         if segue.identifier == "ShowSSOWebView" {
             let vc = segue.destination as! WebViewController
             vc.url = Routes.SSO_URL
+
+            #if OPENSAP
```

Comment:
openWHO has SSO as well...

Reply:
Should we reset the cookies for OpenWHO as well?

Reply:
yes, i guess we should delete all cookies given that this in the scope of  the web view and the app only